waspc/data/Generator/templates/db/schema.prisma

```diff
@@ -10,7 +10,6 @@ datasource db {
 
 generator client {
   provider = "prisma-client-js"
-  output = {=& prismaClientOutputDir =}
```

Comment:
See https://github.com/wasp-lang/wasp/pull/1604 for Prisma-related discussion in the new structure.